analyzer/windows/lib/api/process.py

```diff
@@ -642,6 +682,11 @@ def upload_memdump(self):
             log.error(e, exc_info=True)
             log.error(os.path.join("memory", f"{self.pid}.dmp"))
             log.error(file_path)
-        log.info("Memory dump of process %d uploaded", self.pid)
+        log.info("memory dump of %s uploaded", self)
 
         return True
+
+    def __str__(self):
+        """Get a string representation of this process."""
+        image_name = self.get_image_name() or "???"
+        return f"<{self.__class__.__name__} {self.pid} {image_name}>"
```

Comment:
This method allows us to do `log.info("%s", self)`

Reply:
Ah ok cool - sorry to be a pain!